PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs.SyncLab
+{
+    public class SyncLabShapeStorage : PowerPointPresentation
+    {
+#pragma warning disable 0618
```

Comment:
`The warning disable 0618 is to get the clipboard's content` can u show where's this code?
For this class, from what I see, the warning is caused by many usages of `Globals` and `PowerPointPresentation.Current`